resources/content/schema/admin/admin-auth.json

```diff
@@ -1,46 +1,70 @@
 {
     "authorize" : {
-        "[a-zA-Z]*" : "crud",
-        "[a-zA-Z]*\\..*" : "r",
-
-        ".*\\..*Id" : "cru",
-        ".*\\.data" : "cru",
+    	".*" : "",
+    	
+    	".*\\..*Id" : "r",
+        ".*\\.id" : "r",
+        ".*\\.data" : "r",
         ".*\\.description" : "cru",
         ".*\\.isPublic" : "cru",
-        ".*\\.kind" : "cr",
+        ".*\\.kind" : "r",
         ".*\\.name" : "cru",
         ".*\\.removeTime" : "ru",
-        ".*\\.uuid" : "cr",
-
+        ".*\\.uuid" : "r",
+        ".*\\.accountId" : "r",
+        ".*\\.created" : "r",
+        ".*\\.state" : "r",
+        ".*\\.removed" : "r",
+        ".*\\.transitioning" : "r",
+        ".*\\.transitioningMessage" : "r",
+        ".*\\.transitioningProgress" : "r",
+        
+        "account" : "crud",
+        "account.id" : "cru",
         "account.externalId" : "cru",
         "account.externalIdType" : "cru",
-        "account.defaultCredentialIds" : "cru",
-        "account.defaultNetworkIds" : "cru",
+        "account.kind" : "cr",
```

Comment:
In the spreadsheet, I see no reason why account.kind and account.uuid should be "cr". Seems like they should be "r" according to spreadsheet.

Is there another reason that I'm missing?